shapely/constructive.py

```diff
@@ -188,7 +189,7 @@ def buffer(
 
 @multithreading_enabled
 def offset_curve(
-    geometry, distance, quadsegs=8, join_style="round", mitre_limit=5.0, **kwargs
+    geometry, distance, quadsegs=8, join_style="ROUND", mitre_limit=5.0, **kwargs
```

Comment:
```suggestion
    geometry, distance, quadsegs=8, join_style="round", mitre_limit=5.0, **kwargs
```

(to be consistent with buffer())